packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -155,3 +158,119 @@ export function getNetworkName(
       return 'Unknown'
   }
 }
+
+const SEVEN_DAYS_IN_SECONDS = 7 * 24 * 60 * 60
+
+const mainnetTokenBridge = {
```

Comment:
@spsjvc I had to hardcode these from arbitrum-sdk. Is it possible that our allowed networks - mainnet, nova and one, can be exposed from there itself and used everywhere?

Reply:
NIT: If there are no options but hardcoding, I would use enum here. But ideally, we can import them.

Reply:
they are exported from the sdk so you should be able to use them
```
import {
  l1Networks,
  l2Networks
} from '@arbitrum/sdk/dist/lib/dataEntities/networks'

export const networksList: { [chainId: number]: L1Network | L2Network } = {
  ...l1Networks,
  ...l2Networks
}
```

we probably want to filter out the networks we don't need for the network switch UI